subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/CustomPluginIntegrationTest.groovy

```diff
@@ -151,38 +147,31 @@ class CustomPluginTest {
     @Test
     public void test() {
         def project = ProjectBuilder.builder().build()
-
         project.apply plugin: 'custom'
-
         assert project.custom == 'value'
     }
 }
 """
 
         buildFile << """
-apply plugin: 'groovy'
+plugins {
+    id 'java-gradle-plugin'
+    id 'groovy'
```

Comment:
`groovy-gradle-plugin` also works here, if you want to have just one plugin line